lib/cast/cast_sender.js

```diff
@@ -211,7 +217,12 @@ shaka.cast.CastSender = class {
     // Use static versions of the API callbacks, since the ChromeCast API is
     // static. If we used local versions, we might end up retaining references
     // to destroyed players here.
-    const sessionRequest = new chrome.cast.SessionRequest(this.receiverAppId_);
+    const sessionRequest = new chrome.cast.SessionRequest(this.receiverAppId_,
+        [],
+        // I would like this to be: chrome.cast.timeout.requestSession but the
+        // compiler throws an error that it doesn't exist
+        undefined, 
```

Comment:
I would like this to be: chrome.cast.timeout.requestSession but the compiler throws an error that it doesn't exist